conf/test_full.config

```diff
@@ -34,5 +34,9 @@ params {
 
     // Activate GSEA
     gsea_run = true
-    gsea_gene_sets = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/gene_set_analysis/mh.all.v2022.1.Mm.symbols.gmt'
+    gene_sets_files = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/gene_set_analysis/mh.all.v2022.1.Mm.symbols.gmt'
+
+    // Activate gprofiler2
+    gprofiler2_run = true
+    gprofiler2_token = 'gp__0YRX_1WHk_VjE'
```

Comment:
Unfortunately I haven't found any info on how long exactly a token is available, but it should definitely last for some time (after it expires, I can generate a new one; not ideal of course).
I can also leave it out, then the gene sets will be used, but gprofiler does not find results with the gene sets file and I thought it would be good to have a test config in which gprofiler generates some pathway results.

Reply:
OK, we can't have an ephemeral token (it will almost certainly have finite life) that can just expire and invalidate tests, that's just madness.

Why wouldn't gprofiler find results with just the gene sets? What's the functional difference in what is done with the token?

Reply:
I am really not sure about this, but I suspect that gprofiler is only familiar with those datasources that the server offers (REAC, KEGG, GO etc.); the GMT in the test contains a bunch of hallmark entries and it is possible that gprofiler does not know what to do with those (for that GMT, I also cannot get any results with the [online service](https://biit.cs.ut.ee/gprofiler/gost)).

The token I added is simply from a previous gprofiler run which queried its normal databases, that's why it will find results there. But I'll remove the token. For now, the test will then only produce empty tables (which can of course also happen for an actual analysis).
I can sometime add some additional entries to the GMT in the config so that it will work for both gsea and gprofiler

Reply:
I'm not totally fussed with it using the same gene sets as GSEA. 

How would you run this module in such a way that it would 'query its normal databases' without using the token?

Reply:
I'd give it a GMT that contains some of the sources listed in gprofiler like GO or REAC. Again, it's possible that gprofiler for some reason just happens to not find any enrichment in the current GMT but would be totally fine with some other HALLMARK pathways, I'm not sure and wasn't able to find out for certain why it does not find enrichment; that is just my guess...

Reply:
But you can run gprofiler here **without** supplying the gene sets, right? Weren't you doing that before? And that produced some results? And then it would use just its in-built gene sets, as you did in the UI? 

If I've understood that correctly, you might need to add an additional 'gprofiler_use_gene_sets' flag or somesuch such that it ignores any globally defined gene sets when specifified.

Reply:
Done!

Reply:
I'm not seeing it- could you clarify?

Reply:
That'll be because I forgot to push -.- Sorry, this time it is _actually_ done